docker/server/build.bash

```diff
@@ -29,12 +29,14 @@ function prepare {
     cp -r ${DOCKER_SERVER_DIR}/../../target/repo-*.json ${DOCKER_SERVER_DIR}/target
     # copy over the build zip repos (only 1.6.1 and 1.7)
     cp -r ${DOCKER_SERVER_DIR}/../../target/repo-*.zip ${DOCKER_SERVER_DIR}/target
+    # copy over the packages files
+    (cd ${DOCKER_SERVER_DIR}/../../target; rsync -R */package/* ${DOCKER_SERVER_DIR}/target)
```

Comment:
Does this significantly increase the footprint of the service?

Reply:
Yep, I believe so, we are adding a single file for each package for each DC/OS version. I see no harm though.